docs/1_Getting-Started/2_docker.md

```diff
@@ -0,0 +1,29 @@
+![Docker](../../static/images/docker-logo.png)
```

Comment:
SAA

Reply:
This image only exists in this branch currently and after talking with @luciguci we are going to put this in as is and test the docs site. If changes are needed they will be made after testing the docs site